numba/targets/arrayobj.py

```diff
@@ -1546,6 +1547,80 @@ def array_T(context, builder, typ, value):
     return impl_ret_borrowed(context, builder, typ, res)
 
 
+@generated_jit(nopython=True)
```

Comment:
@esc Would you mind taking a look at this? Importing `generated_jit` fails with a circular import error message:

```
ImportError: cannot import name 'generated_jit' from partially initialized module 'numba.decorators' (most likely due to a circular import) (C:\<snipped>\numba\numba\decorators.py)
```

How do I work around this most elegantly? Copying the `generated_jit` implementation does not seem like the way to go...

Reply:
Ok, so the solution I am using for now is:

```python
def normalize_axis_list(axis, ndim, argname=None, allow_duplicate=False):
    # Stub for overload
    pass

@overload(normalize_axis_list)
def _normalize_axis_list_ovl(axis, ndim, argname=None, allow_duplicate=False):
    # original code
```